packages/graphql/src/types/app.ts

```diff
@@ -0,0 +1,158 @@
+import {
```

Comment:
this file is new and defines all of the custom types for backend usage. the types as added to PrismaJson at the end can be used within the prisma schema as part of the special comments syntax.